notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-1136_M1M3_analysis_stability_tracking.ipynb

```diff
@@ -0,0 +1,704 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    durations = [track.duration for track in tracks_block146]
</pre><p> </p><p>I saw some numbers you posted in the jira ticket (and the discussion with Bruno) about the duration of tracking and got curious if you are calculating the duration or just getting from the TMAEventMaker class. Have you compared the numbers you have with the duration shown in RubinTV (e.g. <a href="https://roundtable.lsst.codes/rubintv-dev/summit/tma/historical/2023-12-20" target="_blank">https://roundtable.lsst.codes/rubintv-dev/summit/tma/historical/2023-12-20</a> or <a href="https://roundtable.lsst.codes/rubintv-dev/summit/tma/historical/2023-12-20" target="_blank">https://roundtable.lsst.codes/rubintv-dev/summit/tma/historical/2023-12-20</a>)?</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/107/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>I have calculated the duration of trackings with the time they start and the time they ends. I didn't know these web pages, but indeed, the duration of their trackings are what I get as well. I don't know why it was said the duration was 30 seconds when both, on the web page and in the calculations it is 42 seconds.</p>